projects/hslayers/src/components/core/core.service.ts

```diff
@@ -142,6 +142,7 @@ export class HsCoreService {
   }
 
   /**
+   * FIXME: statusmanager
```

Comment:
What about this FIXME, @FilipLeitner ? Leave it for another PR?

Reply:
Seems like its not used at all and should be just deleted. Probably I left note there just to give it one more thought...but seems preety straight forward to me now. What you think?

Reply:
No usage of `isAuthorized()` in our codebase any more. Seems safe to delete to me 👍

Reply:
[c24832d](https://github.com/hslayers/hslayers-ng/pull/4014/commits/c24832d71d768eaeef6f6e7847829556a7539fc3)